Source/player.cpp

```diff
@@ -26,6 +26,205 @@
 #include "utils/language.h"
 #include "utils/log.hpp"
 
+namespace
+{
+	using namespace devilution;

```

Comment:
We usually put the anon name space inside of the main one.
```suggestion
namespace devilution {
namespace {
```

Reply:
Also don't indent by namespace, are you running clang-format?